plugin.php

```diff
@@ -10,6 +10,13 @@
 	exit();
 }
 
+define( 'CREODE_BLOCKS_PLUGIN_FOLDER', plugin_dir_path( __FILE__ ) );
```

Comment:
Not sure if this should be defined here. This may be confusing if child themes are used. It implies the the plugin expects all blocks to be registered in the same folder.